daemon/ucrednet.go

```diff
@@ -63,19 +66,36 @@ func ucrednetGetWithInterfaceImpl(remoteAddr string) (ucred *ucrednet, iface str
 			u.Uid = uint32(v)
 		}
 		u.Socket = subs[3]
```

Comment:
Since the group is technically just the `([^;]*)` following the `socket=`, I think for consistency I'll say:
```go
            // group: ([^;]*) - socket path following socket=
```